docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/MySQLIntegrationSuite.scala

```diff
@@ -150,4 +150,91 @@ class MySQLIntegrationSuite extends DockerJDBCIntegrationSuite {
     df2.write.jdbc(jdbcUrl, "datescopy", new Properties)
     df3.write.jdbc(jdbcUrl, "stringscopy", new Properties)
   }
+  
+  test("Write test with SaveMode set to overwrite") {
```

Comment:
Would it be possible to add a table or tables to the dataPreparation function that have column names that are key words? For example:
create table `escaped names` (`key` BIGINT, `long description` TEXT);